### ISSUE TITLE
[0.8.34] Network/Wifi unstable

### ISSUE THREAD

**Issue by Dijker** [athombv/homey#615]
**What is your Homey version (Settings → System)?**
0.8.34 
installed on 12 May, ran severlal days 
**What did you try to achieve?**
use Homey ;-)

*Please include detailed steps how we can reproduce the issue*
Homey is down, not pingable several times a day but comes back online I guess without restarts (Uptime Seconds? still going up!). 

Regarding external monitoring (https://uptimerobot.com/ monitoring https://[Athom_Cloud_ID].homey.athom.com/api/manager/webserver/ping)

Down,"2016-05-16 15:22:26","Internal Server Error","0 hrs, 4 mins"
Up,"2016-05-16 14:45:39",OK,"0 hrs, 36 mins"
Down,"2016-05-16 14:33:18","Internal Server Error","0 hrs, 12 mins"
Up,"2016-05-16 14:28:18",OK,"0 hrs, 5 mins"
Down,"2016-05-16 14:24:58","Internal Server Error","0 hrs, 3 mins"
Up,"2016-05-16 14:13:38",OK,"0 hrs, 11 mins"
Down,"2016-05-16 14:13:07","Connection Timeout","0 hrs, 0 mins"
Up,"2016-05-16 14:07:38",OK,"0 hrs, 5 mins"
Down,"2016-05-16 14:05:17","Internal Server Error","0 hrs, 2 mins"
Up,"2016-05-16 13:34:56",OK,"0 hrs, 30 mins"
Down,"2016-05-16 13:32:35","Internal Server Error","0 hrs, 2 mins"
Up,"2016-05-16 13:12:15",OK,"0 hrs, 20 mins"
Down,"2016-05-16 13:10:54","Internal Server Error","0 hrs, 1 mins"
Up,"2016-05-16 12:59:34",OK,"0 hrs, 11 mins"
Down,"2016-05-16 12:55:13","Internal Server Error","0 hrs, 4 mins"
Up,"2016-05-16 12:34:32",OK,"0 hrs, 20 mins"
Down,"2016-05-16 12:27:11","Internal Server Error","0 hrs, 7 mins"
Up,"2016-05-16 12:22:11",OK,"0 hrs, 5 mins"
Down,"2016-05-16 12:19:50","Internal Server Error","0 hrs, 2 mins"
Up,"2016-05-16 12:14:50",OK,"0 hrs, 5 mins"
Down,"2016-05-16 12:12:29","Internal Server Error","0 hrs, 2 mins"
Up,"2016-05-16 11:09:16",OK,"1 hrs, 3 mins"
Down,"2016-05-16 11:04:55","Internal Server Error","0 hrs, 4 mins"
Up,"2016-05-16 10:59:55",OK,"0 hrs, 5 mins"
Down,"2016-05-16 10:57:35","Internal Server Error","0 hrs, 2 mins"
Up,"2016-05-16 06:46:53",OK,"4 hrs, 10 mins"
Down,"2016-05-16 06:43:32","Internal Server Error","0 hrs, 3 mins"
Up,"2016-05-15 23:43:11",OK,"7 hrs, 0 mins"
Down,"2016-05-15 23:40:51","Internal Server Error","0 hrs, 2 mins"
Up,"2016-05-15 15:00:29",OK,"8 hrs, 40 mins"
Down,"2016-05-15 14:58:09","Internal Server Error","0 hrs, 2 mins"
Up,"2016-05-15 13:47:48",OK,"1 hrs, 10 mins"
Down,"2016-05-15 13:45:27","Internal Server Error","0 hrs, 2 mins"
Up,"2016-05-14 23:50:05",OK,"13 hrs, 55 mins"
Down,"2016-05-14 23:48:44","Internal Server Error","0 hrs, 1 mins"
Up,"2016-05-14 23:43:44",OK,"0 hrs, 5 mins"
Down,"2016-05-14 23:41:24","Internal Server Error","0 hrs, 2 mins"
Up,"2016-05-13 19:45:40",OK,"27 hrs, 55 mins"
Down,"2016-05-13 19:44:39","Internal Server Error","0 hrs, 1 mins"
Up,"2016-05-13 15:14:18",OK,"4 hrs, 30 mins"
Down,"2016-05-13 15:13:18","Internal Server Error","0 hrs, 1 mins"
Up,"2016-05-13 05:10:40",OK,"10 hrs, 2 mins"
Down,"2016-05-13 05:06:19","Internal Server Error","0 hrs, 4 mins"
Up,"2016-05-13 00:34:09",OK,"4 hrs, 32 mins"
Down,"2016-05-13 00:31:48","Internal Server Error","0 hrs, 2 mins"

But also monitoring it's IP address internally (ping -t [Homey's_IP] )
. 
Homey is almost more minutes down than up the last hour. (short after a restart)

**What was the result?**
During "downtime" Homey reacts to 
-RF433 signals (Check433 = on Group button KAKU Do LedRing Pulse Green)
- NFC Card (Homey Says: Sending Logs ) (but times out on sending the log.)

**What did you expect as result?**
Homey to be online!

**Any other remarks**
Send several log after restarting Homey and being offline several times a couple of minutes within an hour or two
20160516 14u55m47 Logs 1691B071A3
&  8FA0DA9059 (don't know the timestamp, if it belongs before or beteeen these ;-) )
&  20160516 15u32m09 Logs 99C16546FE
...

**Comment by Dijker** [comment]
After a one minute power-cycle it happende again. (uptime only some minutes) 
My Ping reply's time out and I don't have access to the web interface of Homey. 
Although that I was able to test KAKU, AND I received a push notification on my phone.
I also was able to send logs (using Flow triggered by NFC card) and get a response (voice and Push)  the logs where send! 
Homey still doesn't reply to ping / http

20160516 16u23m38 -  990AEBB68E
